components/Profile.tsx

```diff
@@ -1,31 +1,47 @@
 import React from "react";
 
 import { useQuery } from "@apollo/react-hooks";
-import { gql } from "apollo-boost";
-import { withApollo } from "../apollo/client";
-
-import { Input, Button } from "antd";
-import Link from "next/link";
-import { Salon, Player } from "@prisma/client";
-
-const Profile = ({
-  data,
-}: {
-  data: { salon: Salon & { players: Player[] }; me: Player };
-}) => {
+import gql from "graphql-tag";
+
+import { useRouter } from "next/router";
+
+const Profile = () => {
+  const player_id =
+    typeof window !== "undefined"
+      ? JSON.parse(localStorage.getItem("salon"))
+      : null;
+  const router = useRouter();
+  const { loading, error, data, client } = useQuery(
+    gql`
```

Comment:
TIL
This is a tagged template literal.
gql is a function called here on a template literal without use of parenthesis.

Reply:
yeah it's pretty cool :)